src/main/java/uk/gov/digital/ho/hocs/workflow/domain/model/forms/HocsFormField.java

```diff
@@ -33,13 +35,26 @@ public static HocsFormField from(FieldDto fieldDto) {
         return new HocsFormField(fieldDto.getComponent(), fieldDto.getValidation(), props);
     }
 
+    public static HocsFormField from(Field field) {
```

Comment:
This is a dupe of the above with just a different method parameter - we could be smarter but as it is deprecated I don't see the point.